### PR TITLE
Improve perf of attachment stream file path upgrade.

### PR DESCRIPTION
**Pull request by charlesmchen**
PTAL @michaelkirk

### CODE REVIEW

**Comment by charlesmchen** [comment]
See: https://github.com/WhisperSystems/Signal-iOS/pull/2256

File: src/Messages/Attachments/TSAttachmentStream.h [hunk 1]
```diff
@@ -52,6 +52,8 @@ NS_ASSUME_NONNULL_BEGIN
 - (CGFloat)audioDurationSecondsWithTransaction:(YapDatabaseReadWriteTransaction *)transaction;
 - (CGFloat)audioDurationSecondsWithoutTransaction;
 
+//- (BOOL)hasUnsavedFilePath;
```
Comment (michaelkirk) [inline src/Messages/Attachments/TSAttachmentStream.h]: did you mean to commit this?

**Review by michaelkirk — APPROVED** [review]
LGTM

File: src/Messages/Attachments/TSAttachmentStream.h (hunk 1 above)
Comment (charlesmchen) [reply src/Messages/Attachments/TSAttachmentStream.h]: Done.

**Pull request merged**